### ISSUE TITLE
Devices not being connectable 

### ISSUE THREAD

**Issue by ggonz20** [momolarson/BLEServiceDiscovery#13]
Hi my name is Jerry. I d like to say thank you for taking the time to write the amazing tutorial online. I am working on a small project that will allow me to control an led light on a raspberry pi.   I installed the libraries and ran the source  code. The app appeared on my device.  The BLE devices app runs.  See below. My main question is how come  I can't connect to any of the devices? I can tap on each of the devices and it takes me to the services menu. My end goal is be able to connect to the device successfully.  
![20210126-191617](https://user-images.githubusercontent.com/37461510/105938741-ae97f380-600c-11eb-8a19-d99f4d299224.jpg)
![20210126-191730](https://user-images.githubusercontent.com/37461510/105938748-b0fa4d80-600c-11eb-9019-741efc7f0e83.jpg)
![20210126-191821](https://user-images.githubusercontent.com/37461510/105938749-b22b7a80-600c-11eb-9079-33e133160247.jpg)

**Comment by momolarson** [comment]
is that an android phone?  If so, there are a lot of differences between them that could be the issue, but, ultimately, that comes from the BLE library.  Have you looked at my other projects which are focused on your specific use case?

The code is here: https://github.com/momolarson/ReactNativeLEDController

and there are links to my articles on how to control LEDs using a raspberry pi.

**Comment by ggonz20** [comment]
Yes it is an android phone. I decided to go that route since I didn't have a Mac machine to develop the app. I tried compiling the LED ![image](https://user-images.githubusercontent.com/37461510/106224430-35c5a280-6198-11eb-80dc-bbffde88c369.jpeg)code but the app stays stuck in this mode.
